lib/resque_spec/matchers.rb

```diff
@@ -185,13 +201,13 @@ def schedule_queue_for(actual)
   chain :at do |timestamp|
     @interval = nil
     @time = timestamp
-    @time_info = "at #{@time}"
+    @time_info = " at #{@time}"
   end
 
   chain :in do |interval|
     @time = nil
     @interval = interval
-    @time_info = "in #{@interval} seconds"
+    @time_info = " in #{@interval} seconds"
```

Comment:
The issues I see if I don't add the blank space are that:

1 - There would be some inconsistency in the `failure_message` method for the `have_scheduled` matcher, given that only this method uses the format of `["expected that #{actual} would not have [#{expected_args.join(', ')}] scheduled", @time_info].join(' ')`, all the others follow the `"expected that #{actual} would be queued with [#{expected_args.join(', ')}]#{@times_info}"` format.
2 - The last line of the `failure_message` will have to be updated to `["expected that #{actual} would not have [#{expected_args.join(', ')}] scheduled", "#{@time_info},", actual_queue_message].join(' ')` so that there is a comma before showing the existent enqueued job in the failure message, and given that there may be times when the @time_info is empty then an extra blank space would appear in those cases.

What do you think?
(Please let me know if I didn't communicate well any of those points.)